File: CS/T686902/ViewModels/MainViewModel.cs
using DevExpress.Mvvm.DataAnnotations;
using System.Collections.ObjectModel;

namespace T686902.ViewModels {
    [POCOViewModel]
    public class MainViewModel {
        protected MainViewModel() {
            Items = new ObservableCollection<Item>();
            for (int i = 0; i < 10; i++)
                Items.Add(new Item() { ID = i, Name = "Item " + i });
        }
        public virtual ObservableCollection<Item> Items { get; set; }
    }

    public class Item {
        public int ID { get; set; }
        public string Name { get; set; }
    }
}

File: VB/T686902/ViewModels/MainViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations
Imports System.Collections.ObjectModel

Namespace T686902.ViewModels
    <POCOViewModel> _
    Public Class MainViewModel
        Protected Sub New()
            Items = New ObservableCollection(Of Item)()
            For i As Integer = 0 To 9
                Items.Add(New Item() With { _
                    .ID = i, _
                    .Name = "Item " & i _
                })
            Next i
        End Sub
        Public Overridable Property Items() As ObservableCollection(Of Item)
    End Class

    Public Class Item
        Public Property ID() As Integer
        Public Property Name() As String
    End Class
End Namespace